jenkins/jobs/kata-containers-2.0-metrics-ubuntu-20-04-PR/config.xml

```diff
@@ -170,12 +179,12 @@ sudo chgrp -R $(id -ng) ${WORKSPACE}</script>
     </org.jenkinsci.plugins.credentialsbinding.impl.SecretBuildWrapper>
     <hudson.plugins.build__timeout.BuildTimeoutWrapper plugin="build-timeout@1.20">
       <strategy class="hudson.plugins.build_timeout.impl.NoActivityTimeOutStrategy">
-        <timeoutSecondsString>900</timeoutSecondsString>
```

Comment:
I assume the timeoutSecondsString is updated by observation?

Reply:
@dborquez yes it is